Gemfile.lock

```diff
@@ -4,94 +4,112 @@ PATH
     shanty (0.3.0)
```

Comment:
:+1: 

I was going to do it separately - can you also close #8 when this is merged?